README.md

```diff
@@ -31,19 +31,124 @@ You are welcome to send your plugin to our contrib by creating a Pull Request!
+
+Example code in examples folder, [rasa_plugin_bot](https://github.com/wechaty/python-wechaty-plugin-contrib/blob/master/examples/rasa_plugin_bot.py)
+
+### 4 Scheduler
+- Author: [@wj-Mcat](https://github.com/wj-Mcat)
+- Description: Say something everyday, like `Scheduler Words`. 
+
+```python
+from wechaty_plugin_contrib import SchedulerPlugin, SchedulerPluginOptions
+#needs more infomations and example code here.
+bot.use(SchedulerPlugin())
+```
+
+### 5 Room_inviter
+- Author: [@markoxu](https://github.com/markoxu)
+- Description: Invite peaple to room according keywords and rules then say hello words.
+
+```python
+from wechaty_plugin_contrib import RoomInviterPlugin, RoomInviterOptions
+from typing import Dict
+
+rules: Dict[MessageMatcher, RoomMatcher] = {
+        MessageMatcher('4群'): RoomMatcher('业主4群'),
+        MessageMatcher('5群'): RoomMatcher('业主5群'),
+}
+
+room_inviter_plugin = RoomInviterPlugin(options=RoomInviterOptions(
+    name='python-wechaty关键字入群插件',
+    rules=rules,
+    welcome = 'welcome join our group.'
+))
+
+bot.use(RoomInviterPlugin())
+```
+
+### 6 Auto_reply
+- Author: [@markoxu](https://github.com/markoxu)
+- Description: AutoReply according keywords. |
+
+Example code in examples folder, [auto-reply-bot](https://github.com/wechaty/python-wechaty-plugin-contrib/blob/master/examples/auto-reply-bot.py)
+
+### 7 Chat_history
+- Author: [@markoxu](https://github.com/markoxu)
+- Description: Save chat history with files and database.
+
+Example code in examples folder, [chat_history_bot](https://github.com/wechaty/python-wechaty-plugin-contrib/blob/master/examples/chat_history_bot.py)
+
+### 8 Github_webhook
+- Author: [@wj-Mcat](https://github.com/wj-Mcat)
+- Description: Github webhook, hook infomation to peaple or room when projects were updated. 
+
+```python
+# did't test, needs more infomations and example code here.
+from wechaty_plugin_contrib import GithubHookItem, GithubWebhookOptions
+bot.use(GithubHookItem())
+```
+
+### 9 GitlabEvent
+- Author: [@wj-Mcat](https://github.com/wj-Mcat)
+- Description: Gitlab webhook, send gitlab event infomation to peaple or room when projects were updated.
+
+```python
+# did't test, needs more infomations and example code here.
+from wechaty_plugin_contrib import GitlabEventPlugin, GitlabEventOptions
+
+bot.use(GitlabEventPlugin())
+```
+
+### 10 Example code How to find room or contact.
+- Author: [@wj-Mcat](https://github.com/wj-Mcat)
+- Description: Example code How to find room or contact, then do something.
+
+Example code in examples folder, [Example of Finders](https://github.com/wechaty/python-wechaty-plugin-contrib/blob/master/examples/finders.py)
+
+
+### 12 Use_two_plugins
+- Author: [@xinyu3ru](https://github.com/xinyu3ru)
+- Description: Example code How to use two or more plugins. 
+
+Example code in examples folder, [use_two_plugins](https://github.com/wechaty/python-wechaty-plugin-contrib/blob/master/examples/use_two_plugins.py)
```

Comment:
Please get the author of plugin from the pull request or forced commit. Or you can see the history of plugin file not the example code file.